### PR TITLE
fix functions with empty cells

### PR DESCRIPTION
**Pull request by llllllllll**
addresses #73

### CODE REVIEW

File: cloudpickle/cloudpickle.py [hunk 1]
```diff
@@ -896,6 +897,24 @@ def _gen_ellipsis():
...
+
+def _get_cell_contents(cell):
+    try:
+        return cell.cell_contents
+    except ValueError:
+        # sentinel used by ``_fill_function`` which will leave the cell empty
+        return _empty_cell_value
+
+
+@object.__new__
```
Comment (pitrou) [inline cloudpickle/cloudpickle.py]: Could this be less cryptic? :-)

Comment (llllllllll) [inline cloudpickle/cloudpickle.py]: I have wrapped this in a named function with a docstring.

Idk if this is better.

File: cloudpickle/cloudpickle.py [hunk 2]
```diff
@@ -896,6 +897,40 @@ def _gen_ellipsis():
...
+
+def _get_cell_contents(cell):
+    try:
+        return cell.cell_contents
+    except ValueError:
+        # sentinel used by ``_fill_function`` which will leave the cell empty
+        return _empty_cell_value
+
+
+def instance(cls):
```
Comment (pitrou) [inline cloudpickle/cloudpickle.py]: Ha :-)

Comment (rgbkrk) [inline cloudpickle/cloudpickle.py]: Should this just go back to `@object.__new__`?

Comment (pitrou) [inline cloudpickle/cloudpickle.py]: No, it's ok to me :-)

**Pull request merged**